mep/people/admin.py

```diff
@@ -133,6 +133,34 @@ class Media:
         js = ('admin/js/prepopulate.js', )
 
 
+class PersonTypeListFilter(admin.SimpleListFilter):
+    '''Filter that for :class:`~mep.people.models.Person` that can distinguish
+    between people who are creators of books vs. those who are library members.
+    '''
+    # human-readable title for filter
+    title = 'person type'
+
+    # this gets used in the URL as a query param
+    parameter_name = 'person_type'
+
+    def lookups(self, request, model_admin):
+        # option tuples: left is query param name and right is human-readable name
+        return (
+            ('creator', 'creator'),
```

Comment:
For consistency with other filters, I think we should capitalize the labels here